conf/mesos-site.xml

```diff
@@ -19,54 +19,84 @@
 <configuration>
   <property>
     <name>mesos.hdfs.data.dir</name>
+    <description>The primary data directory in HDFS</description>
     <value>/tmp/hdfs/data</value>
   </property>
   
   <property>
     <name>mesos.hdfs.secondary.data.dir</name>
+    <description>The secondary data directory in HDFS</description>
```

Comment:
What is the secondary data directory though? Seems like this is somehow tied to dfs.domain.socket.path
(not a merge blocker, just a documentation ambiguity)